scico/flax/train/traversals.py

```diff
@@ -7,89 +7,14 @@
 
 """Functionality to traverse, select and update model parameters."""
```

Comment:
Functionality to traverse, select, and update model parameters.

Reply:
Edited.